netplan/cli/utils.py

```diff
@@ -40,16 +42,22 @@ def get_generator_path():
 
 
 def is_nm_snap_enabled():
-    return subprocess.call(['systemctl', '--quiet', 'is-enabled', NM_SNAP_SERVICE_NAME], stderr=subprocess.DEVNULL) == 0
+    env = dict(LC_ALL='C', PATH=os.environ.get('PATH', FALLBACK_PATH))
+    return subprocess.call(['systemctl', '--quiet', 'is-enabled', NM_SNAP_SERVICE_NAME],
+                           env=env, stderr=subprocess.DEVNULL) == 0
```

Comment:
I'm wondering if it would suffice setting these variables only once via os.environ. Something like the below in the entry point of the Python code:

`os.environ.update({'LC_ALL': 'C', 'PATH': os.environ.get('PATH', '/usr/bin:/snap/bin')})`

Reply:
Yes, moving it all into `cli/core.py` in the `class Netplan` constructor makes lots of sense. Let me test if this still works as expected (it should).